blazor/diagram/scroll-settings.md

```diff
@@ -64,6 +64,51 @@ You can programmatically change the scroll offsets at runtime by using the exter
 }
 ```
 
+## ScrollChanged Event
+
+The Diagram control provides the following event for the scroll settings.
+
+| Event Name | Event Type | Description |
+| -------- | -------- | -------- |
+| [ScrollChanged](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.SfDiagramComponent.html#Syncfusion_Blazor_Diagram_SfDiagramComponent_FixedUserHandleClick) | [ScrollChangedEventArgs](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.FixedUserHandleClickEventArgs.html) | Triggered when scrollbar is updated. |
```

Comment:
| The [ScrollChanged](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.SfDiagramComponent.html#Syncfusion_Blazor_Diagram_SfDiagramComponent_FixedUserHandleClick) | [ScrollChangedEventArgs](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.FixedUserHandleClickEventArgs.html) | Triggered when the scrollbar is updated. |